src/packages/cli/src/__tests__/integration/mssql/__database.ts

```diff
@@ -1,44 +1,45 @@
-const sql = require('mssql')
-const url = require('url');
+import mssql from 'mssql'
+import url from 'url'
 
 import { Context, Input } from '../../__helpers__/integrationTest'
 
 export const database = {
   name: 'sqlserver',
   datasource: {
-    url: (ctx) => getConnectionInfo(ctx).connectionString,
+    url: ctx =>
+      `sqlserver://localhost:1433;database=master_tests;user=SA;password=Prisma1-prisma;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT`,
```

Comment:
The format of this URL is not the same as the one needed by `mssql`, so we'l have to likely add another env var

Reply:
True.